git-cliff-core/src/commit.rs

```diff
@@ -125,18 +125,20 @@ pub struct Commit<'a> {
 	pub merge_commit:  bool,
 	/// Arbitrary data to be used with the `--from-context` CLI option.
 	pub extra:         Option<Value>,
+	/// Remote metadata of the commit.
+	pub remote:        Option<crate::remote_contributor::RemoteContributor>,
 	/// GitHub metadata of the commit.
 	#[cfg(feature = "github")]
-	pub github:        crate::remote::RemoteContributor,
+	pub github:        crate::remote_contributor::RemoteContributor,
```

Comment:
I moved the `RemoteContributor` from the `remote` module because the `remote` module is under the `remote` feature flag.